heft-plugins/heft-isolated-typescript-transpile-plugin/src/SwcIsolatedTranspilePlugin.ts

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { availableParallelism } from 'node:os';
```

Comment:
You can get this from HeftConfiguration now

Reply:
It isn't, but I added that.